packages/core/src/JWEditor.ts

```diff
@@ -122,6 +135,23 @@ export class JWEditor {
             this.debugger = new OwlUI(this);
             this.debugger.addPlugin(DevTools);
         }
+        if (config.shortcuts) {
+            const shortcutWithPriority = config.shortcuts.map(shortcut => ({
+                priority: 1,
+                ...shortcut,
+            }));
```

Comment:
Can't we just have these ones override the others, bypassing the need for setting a `priority` ?

Reply:
I don't like to override the other because for multiples reason:
- We lose information that might be useful later (we could have an interface that shows the default shortcuts as well as the overridden shortcuts)
- If we add a plugin after loading the config, it will not work without extra work